pkg/kubecost/bingen.go

```diff
@@ -71,4 +71,4 @@ package kubecost
 // @bingen:generate:AuditSetRange
 // @bingen:end
 
-//go:generate bingen -package=kubecost -version=15 -buffer=github.com/opencost/opencost/pkg/util
+//go:generate bingen -package=kubecost -version=18 -buffer=github.com/opencost/opencost/pkg/util
```

Comment:
I'm pretty sure you don't have to change the version here -- this only changes the version for things that don't specify their own version, like `Window`.